packages/syft/src/syft/node/node.py

```diff
@@ -1141,14 +1143,41 @@ def handle_api_call_with_unsigned_result(
         return result
 
     def add_action_to_queue(
```

Comment:
Q: currently is the add_action_to_queue only for UserCode objects?

Reply:
Mostly for any kind of queue messages that involve an Action Object.